internal/install/_static/terraform_deployer_run.sh

```diff
@@ -1,7 +1,9 @@
-#!sh
+#!/usr/bin/env bash
 
 set -euxo pipefail
```

Comment:
Let's drop `-x` to have a clean nice output in container logs. This script doesn't contain tangled logic which may cause issues.